PR #21 Reformatted (#128)
mongodb/docs-meta: source/style-guide/markup/directives/tables.txt

================
Table Directives
================

Use tables to format related information.

> In Snooty, you can only use the ``list-table`` directive to build tables.

Reviewer: **Style:** See [Misplaced Modifiers](https://www.mongodb.com/docs/meta/style-guide/writing/clarify-modifiers/#misplaced-modifiers).

```suggestion
In Snooty, you can use only the ``list-table`` directive to build tables.
```

The ``list-table`` directive generates a formatted table from a two-level

> bullet list.

Reviewer: ```suggestion
bulleted list.
```

> Learn more about the ``list-table`` directive frmo the

Reviewer: **Style:** See [Link and Cross-Reference Text](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#link-and-cross-reference-text).

```suggestion
To learn more about the ``list-table`` directive, see the
```

Author: I have issues with this style guide item, but happy to make this change for now.
